src/smc-util-node/test/misc_node-test.coffee

```diff
@@ -48,4 +48,25 @@ describe "sanitizing HTML", ->
+
+describe 'check sales tax work', ->
+    sales_tax = misc_node.sales_tax
+
+    it 'knows about 98122', ->
+        expect(sales_tax('98122')).toBe 0.10100
```

Comment:
smc-util/test might be that way, but that was mostly written by @haraldschilly as his first thing on SMC long ago, and conflicts with our codebase style.   We don't really have time to worry about such stuff though, so forgot about this for now.